PR-App/PR-App/Services/FirebaseManager.swift

```diff
@@ -11,15 +11,18 @@ import FirebaseAuth
 import FirebaseStorage
 import FirebaseDatabase
 
+// you could make several classes (depending on the use case) for Firebase tasks, and provide a protocol for FirebaseManager that would make code testable.
```

Comment:
@StemCll is there any chance that you can show me how to make this properly with several classes?